IronFoundry.Container.Test/ContainerServiceTests.cs

```diff
@@ -227,6 +227,19 @@ public void CreateShouldAddToTheList()
                 }
             }
 
+            public class GetContainerHandles : WithContainer
+            {
+                [Fact]
+                public void ShouldReturnAllHandles()
+                {
+                    var handles = Service.GetContainerHandles();
+                    Assert.Collection(handles,
+                        x => Assert.Equal(Container.Handle, x)
```

Comment:
Should probably check the count of handles returned.
This looks like it just runs the assert over every item in the collection but what if GetContainerHandles returned an empty list?